lib/tasks/reindex_best_bets.rake

```diff
@@ -0,0 +1,23 @@
+desc "Resend all local queries to rummager to be reindexed"
+task reindex_best_bets: :environment do
+  message = <<-MSG
+    Rebuilding the elasticsearch index will just resend all locally stored best bets across to
+    be insertedin to elasticsearch. This means that any orphanded entried in the elasticsearch
+    index would still exist. This can be avoided by running the below rake task on Rummage before
+    rebuilding the index.
+
+    bundle exec rake rummager:switch_to_empty_index RUMMAGER_INDEX=metasearch
```

Comment:
Is it safe to run this? It seems like quite a drastic thing to do even if it's temporary. How long does the repopulation take, and how bad are the search results when the best bets are missing?